_std/macros/mob_properties.dm

```diff
@@ -161,6 +161,11 @@ To remove:
 	SEND_SIGNAL(target, COMSIG_MOB_PROP_INVISIBILITY, old_val); \
 	} while(0)
 
+#define PROP_UPDATE_SIGHT(target, prop, old_val) do {\
+	if(!istype(target, /mob/living)) return; \
+	var/mob/living/_M = target; \
+	_M.lifeprocesses[/datum/lifeprocess/sight].process(); \
```

Comment:
tmk there should be no instance where this lifeprocess does not exist (outside of disposing)